internal/cmd/build/go.mod

```diff
@@ -1,6 +1,8 @@
 module go.temporal.io/sdk/internal/cmd/build
 
-go 1.20
+go 1.21
+
+toolchain go1.22.4
```

Comment:
We don't want to specify a toolchain since it will override our CI that trys to test the SDK on all the versions of Go we support.

Reply:
Yeah didn't realize GoLand added this.